tests/basilisp/contrib/nrepl_server_test.lpy

```diff
@@ -559,6 +575,40 @@
             {:id @id* :ns "abc.other" :value "\"abc\""}
             {:id @id* :ns "abc.other" :status ["done"]})
 
+          (client-send! client {:id (id-inc!) :ns "abc.xyz":op "eval" :code "(abc)"
+                                :file filename
+                                :line 6 :column 2})
```

Comment:
```suggestion
          (client-send! client {:id (id-inc!) :ns "abc.xyz" :op "eval" :code "(abc)"
                                :file filename
                                :line 6 :column 2})
```

I guess I wouldn't hate if this was each key on its own line, but if not then can you at least add a space between this string and keyword.

Reply:
Updated to add a space. Apologies for not placing each key on its own line; the rest of the test file follows an ad-hoc style for visual clarity and space conservation, which might seem inconsistent otherwise. Perhaps we could look into a tool similar to `black` in Python for automatic code formatting?

One tool that comes to mind is [cljfmt](https://github.com/weavejester/cljfmt), which could have caught this with its `:insert-missing-whitespace?` option and even better would have split the keys into their own lines using `:split-keypairs-over-multiple-lines?`.

Reply:
I like the idea and as you know I've configured it for the Python code. However, I recently surveyed the Clojure formatting tool space for just this purpose and found myself unsatisfied with any of the options. I primarily write Clojure in Emacs (Spacemacs) and have always found that I prefer its implicit formatting style (with alignment), but none of the tools would enforce that style for me without other drawbacks or difficulties.

Reply:
> I like the idea and as you know I've configured formatting checks for the Python code. 

Indeed, this is why I mentioned it. It's very useful.

> I primarily write Clojure in Emacs (Spacemacs) and have always found that I prefer its implicit formatting style (with alignment), but none of the tools would enforce that style for me without other drawbacks or difficulties.

Out of curiosity, which specific alignment setting do you find missing in `cljfmt`? It seems fairly configurable, or is it more the overall alignment style that’s tricky to set up?